pkg/users/password-dialogs.js

```diff
@@ -171,33 +145,16 @@ function SetPasswordDialogBody({ state, errors, change }) {
                 <TextInput className="check-passwords" type="password" id="account-set-password-old"
                            value={password_old} onChange={value => change("password_old", value)} />
             </FormGroup> }
-
-            <FormGroup label={_("New password")}
-                       helperTextInvalid={errors && errors.password}
-                       validated={(errors && errors.password) ? "error" : "default"}
-                       fieldId="account-set-password-pw1">
-                <TextInput className="check-passwords" type="password" id="account-set-password-pw1"
-                           value={password} onChange={value => change("password", value)} />
-            </FormGroup>
-
-            <FormGroup label={_("Confirm new password")}
-                       helperTextInvalid={errors && errors.password_confirm}
-                       validated={(errors && errors.password_confirm) ? "error" : "default"}
-                       fieldId="account-set-password-pw2">
-                <TextInput type="password" id="account-set-password-pw2"
-                           value={password_confirm} onChange={value => change("password_confirm", value)} />
-                <div>
-                    <div id="account-set-password-meter" className={"progress password-strength-meter " + password_strength}>
-                        <div className="progress-bar" />
-                        <div className="progress-bar" />
-                        <div className="progress-bar" />
-                        <div className="progress-bar" />
-                    </div>
-                    <div>
-                        <span id="account-set-password-meter-message" className="help-block">{password_message}</span>
-                    </div>
-                </div>
-            </FormGroup>
+            <PasswordFormFields password={password}
+                                password_confirm={password_confirm}
+                                password_label={_("New password")}
+                                password_confirm_label={_("Confirm new password")}
+                                password_strength={password_strength}
+                                password_message={password_message}
+                                error_password={errors && errors.password}
+                                error_password_confirm={errors && errors.password_confirm}
+                                idPrefix="account-set-password"
+                                change={change} />
```

Comment:
(commenting here as I cannot comment down there)
On line 193 there is `Excellent password` but I don't think we ever see this string. Just git grep for this string. Possibly not related to this PR, then fine for further cleanup. Mostly concert as we used to show this string when you reached quality of 100.